src/scope/scope.js

```diff
@@ -538,6 +538,91 @@ export default class Scope {
     return Promise.all(versionDependenciesArr.map(versionDependencies => versionDependencies.toConsumer(this.objects)));
   }
 
+  /**
+   * Remove or deprecate single component
+   * @removeComponent - boolean - true if you want to remove component
+   */
+  async removeSingle(bitId: BitId, removeComponent: boolean): Promise<string> {
```

Comment:
split this to 2 different functions remove / deprecate

Reply:
done